mappings/net/minecraft/client/entity/particle/RainSpalshParticle.mapping

```diff
@@ -0,0 +1,5 @@
+CLASS net/minecraft/class_260 net/minecraft/client/entity/particle/RainSpalshParticle
```

Comment:
typo?
```suggestion
CLASS net/minecraft/class_260 net/minecraft/client/entity/particle/RainSplashParticle
```

Reply:
Yeah, that’s a typo. I need to get back to this after being distracted for a few months by game engines and c++.